src/controllers/sync-controller.ts

```diff
@@ -143,10 +361,67 @@ export class SyncController implements ReactiveController {
     }
   }
 
+  /**
+   * Register periodic background sync if capabilities are available and settings allow
+   * Phase 1.3: App initialization periodic sync registration
+   */
+  async #registerPeriodicBackgroundSyncIfAvailable(): Promise<void> {
+    // Only proceed if we have both service worker and periodic background sync APIs
+    if (!this.#serviceWorkerReady || !this.#pwaCapabilities.periodicBackgroundSync) {
+      if (this.#serviceWorkerReady) {
+        DebugService.logInfo('sync', 'Periodic background sync API not available - falling back to timer-based sync when app is visible');
+      }
+      return;
+    }
+
+    try {
+      // Wait for settings to be available
+      if (this.isSettingsLoading) {
+        // Listen for settings to become available
+        const settings = await new Promise<AppSettings | null>((resolve) => {
```

Comment:
Since `#settingsQuery` is already reactive, can't we just have this method called whenever the value of `#settingsQuery` changes rather than using Promise-based polling as we're doing here?